lib/plugins/aws/package/compile/functions.js

```diff
@@ -629,6 +629,30 @@ class AwsCompileFunctions {
       }
     }
 
+    if (functionObject.loggingConfig) {
+      const loggingConfig = {};
+
+      if (functionObject.loggingConfig.applicationLogLevel) {
+        loggingConfig.ApplicationLogLevel = functionObject.loggingConfig.applicationLogLevel;
+      }
+
+      if (functionObject.loggingConfig.logFormat) {
+        loggingConfig.LogFormat = functionObject.loggingConfig.logFormat;
+      }
+
+      if (functionObject.loggingConfig.logGroup) {
+        loggingConfig.LogGroup = functionObject.loggingConfig.logGroup;
```

Comment:
By default, Serverless also provisions the Log Group itself as part of it's cloudformation stack. At that stage, the custom log group name also needs to be checked.